crates/matrix-sdk-crypto/src/identities/manager.rs

```diff
@@ -2023,4 +2023,269 @@ pub(crate) mod tests {
 
```

Comment:
This commit: "crypto: Verified identity changes TDD" - I would say that the tests for something could be in the same commit as the code itself, so no need for a separate commit here.

If you want a separate commit I suggest "crypto: Tests for verified_latch" or similar.